Bottom: dd712c919689148521a5619ff798cbec70933c14
Top:    d29dad9be923e273ae126fa1609a8fe8d1726c9d
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-29 16:39:45 +0800

Refresh of tuntap-napi

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 5f67071..ed1a70d 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -647,7 +647,7 @@ static int tun_poll(struct napi_struct *napi, int budget)
 	struct sk_buff *skb;
 	unsigned int received = 0;
 
-	while ((skb = skb_dequeue(input_queue))) {
+	while ((skb = tun_dequeue(input_queue))) {
 		netif_receive_skb(skb);
 		if (++received >= budget)
 			return received;
